sdk/core/azure-core/CMakeLists.txt

```
@@ -22,6 +22,10 @@ az_vcpkg_integrate()
 
 find_package(Threads REQUIRED)
 
+# Allow building azure-core without OpenSSL dependency and hence without Crypto-functions
+# This is a work-around for anyone consuming azure-core without a need for Crypto like the cognitive services.
+option(BUILD_CRYPTO "Build the cryto namespace. Build will depend on OpenSSL." ON)
```

I thought we are getting rid of the openssl dependency all together, no?

```suggestion
option(BUILD_CRYPTO "Build the crypto namespace. Build will depend on OpenSSL." ON)
```